x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -118,9 +112,14 @@ export function WorkspacePanelWrapper({
     warningMessages.push(...requestWarnings);
   }
   return (
-    <EuiPageTemplate direction="row" grow={true} offset={0} minHeight={0} responsive={[]}>
+    <EuiPageTemplate direction="column" grow={false} offset={0} minHeight={0}>
       {!(isFullscreen && (autoApplyEnabled || warningMessages?.length)) && (
-        <EuiPageSection grow={false} paddingSize="none" color="transparent">
+        <EuiPageTemplate.Section
+          grow={false}
+          paddingSize="none"
+          color="transparent"
+          restrictWidth={false}
```

Comment:
With `restrictWidth={false}` moved to the parent `EuiPageTemplate` component, this can be removed.

```suggestion
```